src/Sarif.Converters/FortifyConverter.cs

```diff
@@ -147,8 +147,17 @@ public static Result ConvertFortifyIssueToSarifIssue(FortifyIssue fortify)
                     {
                         Locations = new List<AnnotatedCodeLocation>
                         {
-                            new AnnotatedCodeLocation { PhysicalLocation = source, Essential = true },
-                            new AnnotatedCodeLocation { PhysicalLocation = primaryOrSink, Essential = true }
+                            new AnnotatedCodeLocation
+                            {
+                                PhysicalLocation = source,
+                                Importance = AnnotatedCodeLocationImportance.Essential
+                            },
+
+                            new AnnotatedCodeLocation
+                            {
+                                PhysicalLocation = primaryOrSink,
+                                Importance = AnnotatedCodeLocationImportance.Essential
```

Comment:
I agree, but here I was just directly translating the "`Essential = true`" on the left-hand side of the diff to "`Importance = AnnotatedCodeLocationImportance.Essential``" on the right-hand side.

I think you're saying that the "`Essential = true`" shouldn't have been there in the first place, and I agree with that.

I'm going to write myself a note and take care of this in a separate checkin.